app/src/hooks/useBioHubApi.ts

```diff
@@ -117,13 +123,39 @@ export const useBiohubApi = () => {
     return data;
   };
 
+  /**
+   * Fetch the media list.
+   *
+   * @return {*} {Promise<IMedia[]>}
+   */
+
+  const getMediaList = async (projectId: string): Promise<IMedia[]> => {
+    //const api = await apiPromise;
+
+    const { data } = await api.get('/api/projects/' + projectId + '/artifacts/list');
```

Comment:
This is maybe a subjective thing, but I think its nicer if we use template strings:
```Typescript
const { data } = await api.get(`/api/projects/${projectId}/artifacts/list`);
```